.storybook/preview.js

```diff
@@ -0,0 +1,15 @@
+import { withGlobalStyles } from './custom/withGlobalStyles';
+import { withThemesProvider } from 'storybook-addon-styled-component-theme';
+import { defaultTheme } from '../src/defaultTheme';
+
+export const parameters = {
+  backgrounds: {
+      values: [
```

Comment:
Formatting here too.

Reply:
ESLint / Prettier don't touch these files for some reason? I assume .storybook is ignored somewhere but I can't see where. I have manually adjusted this formatting here